src/shacl-shapes.xsl

```diff
@@ -36,7 +36,7 @@
     <xsl:import href="shacl-shape-lib/shacl-shape-selectors.xsl"/>
     <xsl:import href="shacl-shape-lib/elements-shacl-shape.xsl"/>
     <xsl:import href="shacl-shape-lib/connectors-shacl-shape.xsl"/>
-    <xsl:import href="../config/config-proxy.xsl"/>
+    <!--<xsl:import href="../config/config-proxy.xsl"/>-->
```

Comment:
how do you provide the config? through CLI parameters?

Reply:
This config-proxy.xsl is already loaded multiple times via other imported file.